support-workers/src/main/scala/com/gu/support/workers/lambdas/SendAcquisitionEvent.scala

```diff
@@ -34,29 +34,38 @@ class SendAcquisitionEvent(serviceProvider: ServiceProvider = ServiceProvider)
   ): FutureHandlerResult = {
     SafeLogger.info(s"Sending acquisition event to ophan: ${state.toString}")
     // Throw any error in the EitherT monad so that in can be processed by ErrorHandler.handleException
-    services.acquisitionService.submit(state).fold(errors => throw AnalyticsServiceErrorList(errors), _ => HandlerResult(Unit, requestInfo))
+    services.acquisitionService.submit(
+      SendAcquisitionEventStateAndRequestInfo(state, requestInfo)
+    ).fold(
+      errors => throw AnalyticsServiceErrorList(errors), _ => HandlerResult(Unit, requestInfo)
+    )
   }
 }
 
+case class SendAcquisitionEventStateAndRequestInfo(
```

Comment:
minor: I was going to complain about the name of this class but then I have to propose something better..
maybe AcquisitionBuildData, even though it's kinda meaningless at least it will still make sense if we decide to add or remove something and also it's shorter.
Just a suggestion, feel free to ignore it 100%
also, sorry this comment should have come before my other one